src/pages/assessment/Assessment.tsx

```
@@ -0,0 +1,408 @@
+        onClose={handleCloseModal}
+        onSubmit={handleSubmitAssessment}
+        mode={modalMode}
+        isLoading={discoverySourcesContext.isCreatingAssessment}
+        selectedEnvironment={null}
+      />
+
+      <UpdateAssessment
+        isOpen={isUpdateModalOpen}
+        onClose={handleCloseUpdateModal}
+        onSubmit={handleConfirmUpdate}
+        name={(selectedAssessment as any)?.name || ''}
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)